sys/include/checksum/crc16_ccitt.h

```diff
@@ -39,6 +36,8 @@ extern "C" {
 
 /**
  * @brief           Update CRC16-CCITT
+ * @deprecated      Use @ref crc16_ccitt_false_update instead. 
+ * Will be removed after 2023.01 release.
```

Comment:
```suggestion
 * @deprecated      Use @ref crc16_ccitt_false_update instead.
 *                  Will be removed after 2023.01 release.
```

Reply:
Why is this called `crc16_ccitt_false_update` btw, while the other functions for ccitt-aug are called `crc16_ccitt_aug_...`?

Reply:
That's correct. ccitt-false and ccitt-aug only differ in the starting seed. When in doubt, the tests show of this behaviour and how to use it. 
I thought about having a separate function with the suiting name just as a wrapper for naming clarity - but was to shy to do so.

Reply:
> Oh, could I move the wrapper into the header file?

You _should_. Otherwise you don't get much advantage from the inline ;-)